### PR TITLE
JavaScript capstone

### PR DESCRIPTION
**Pull request by a-laarabi**
# What?
> In this pull request, I @a-laarabi  and @tarek102 created a Webapp that displays movies and their descriptions, also you can like a movie or add a comment for a specific movie.

## Highlights:
- there are no linter errors.
- We used the correct Gitflow.
- We documented our work in a professional way.
- We Followed the list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).
- We use ES6 modules, with import and export.
- We chose [TVmaze API](https://www.tvmaze.com/api) to base the development of the webapp around it.
- We set up our project with webpack and JavaScript testing library (Jest).
- We created a specific function to calculate the movies and the comments displayed.
- We created testing files to test our functions.

## Screen shoot for the testing files 

Capture d’écran 2022-08-06 à 02 58 00

You can open the index by typing the command npm start in the terminal or by opening directly the index.html in the dist directory.

## You can see our [descriptive video](https://youtu.be/-AETBh-FtCg) for this project

Cheer and happy reviewing 🥇

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,58 @@
-# JavaScript-capstone
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# JavaScript group capstone
+
+
+In this project I have used linter for HTML and CSS and Javascript, to make sure there are no linter errors on the code.
```
Comment (Bondok6) [inline README.md]: - [ ] Kindly, add a more descriptive title and description for your project to make a professional `README.md` file 👌
> For Example 💡

> Movies App
> In this project, We Created a webapp that displays movies and their descriptions ........

File: README.md [hunk 2]
```diff
@@ -1 +1,58 @@
...
+## Built With
+
+- HTML
+- CSS
+- Javascript
+
+
+## Live Demo (if available)
+
+[Demo](https://github.com/tarek102/JavaScript-capstone//dist/)
```
Comment (Bondok6) [inline README.md]: - [ ] This link is broken, please fix it or remove it from your `README.md`  👌

File: README.md [hunk 3]
```diff
@@ -1 +1,58 @@
...
+
+
+## Getting Started
+
+To successfully utilize this project first of all you need to clone or download the repository. Next, open the index.html file to check the website’s main page. If you want to add or edit the files you can do that by simply opening the file folder in VS code.
+
```
Comment (Bondok6) [inline README.md]: - [ ] Great job adding the `Getting Started` section 💯. However,  I suggest adding more instructions about how to run your project, so I suggest adding `npm install` to install all dependencies, `npm run start` to run the project, and  `npm run test` to run the test 👌

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @a-laarabi  and @tarek102  🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!



## Suggested changes

- [ ] According to the requirements, you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:

image 

- [ ] _Check the comments under the review._

## To Highlight
- Your project is professional ✅🔥
- Your repo is professional ✅
- Tests are passing ✅
- No linter errors ✅

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by MahmoudBakr23 — APPROVED** [review]
Hi team😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done with the presentation video
- Descriptive README file

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**